internal/exec/util/passwd.go

```diff
@@ -171,16 +178,22 @@ func (u Util) AuthorizeSSHKeys(c types.PasswdUser) error {
 		if !strings.HasSuffix(ks, "\n") {
 			ks = ks + "\n"
 		}
-
+		var path string
 		if distro.WriteAuthorizedKeysFragment() {
-			err = writeAuthKeysFile(usr, filepath.Join(usr.HomeDir, ".ssh", "authorized_keys.d", "ignition"), []byte(ks))
+			path = filepath.Join(usr.HomeDir, ".ssh", "authorized_keys.d", "ignition")
+			err = writeAuthKeysFile(usr, path, []byte(ks))
 		} else {
-			err = writeAuthKeysFile(usr, filepath.Join(usr.HomeDir, ".ssh", "authorized_keys"), []byte(ks))
+			path = filepath.Join(usr.HomeDir, ".ssh", "authorized_keys")
+			err = writeAuthKeysFile(usr, path, []byte(ks))
 		}
-
 		if err != nil {
 			return fmt.Errorf("failed to set SSH key: %v", err)
 		}
+		journal.Send(fmt.Sprintf("wrote ssh authorized keys file for user: %s", c.Name), journal.PriInfo, map[string]string{
+			"IGNITION_USER_NAME": c.Name,
+			"IGNITION_DIR_PATH":  path,
```

Comment:
Hmm, why `DIR_PATH` here? Isn't this a path to the file itself? How about just `IGNITION_PATH`?

Reply:
Fixed